pkg/client/observatorium/observability_config.go

```diff
@@ -118,11 +120,31 @@ func (c *ObservabilityCloudWatchLoggingConfig) setDefaults() {
 	}
 }
 
+func (c *ObservabilityCloudWatchLoggingConfig) GetEnterpriseCredentials(orgID string) *ObservabilityCloudwatchLoggingConfigCredentials {
+	credentials := &ObservabilityCloudwatchLoggingConfigCredentials{}
+	idx, enterpriseCredential := arrays.FindFirst(c.EnterpriseCredentials, func(enterpriseCredential ObservabilityEnterpriseCloudwatchLoggingConfigCredentials) bool {
+		return orgID == enterpriseCredential.OrgID
+	})
+	if idx != arrays.ElementNotFound {
+		credentials.AccessKey = enterpriseCredential.Credentials.AccessKey
+		credentials.SecretAccessKey = enterpriseCredential.Credentials.SecretAccessKey
+
+		return credentials
+	}
+
+	return nil
```

Comment:
```suggestion
	if idx == arrays.ElementNotFound {
		return nil
	}
	credentials.AccessKey = enterpriseCredential.Credentials.AccessKey
	credentials.SecretAccessKey = enterpriseCredential.Credentials.SecretAccessKey

	return credentials
```

Reply:
:+1: I find return early on errors / edge cases is simpler to follow